maxminddb/extension.pyi

```diff
@@ -6,22 +6,24 @@ This module contains the C extension database reader and related classes.
 
 """
 
+# pylint: disable=E0601,E0602
 from ipaddress import IPv4Address, IPv6Address
 from os import PathLike
-from typing import Any, AnyStr, Dict, IO, List, Optional, Tuple, Union
-from maxminddb import MODE_AUTO
+from typing import IO, Any, AnyStr, Dict, List, Optional, Tuple, Union
+
 from maxminddb.types import Record
 
 class Reader:
-    """
-    A C extension implementation of a reader for the MaxMind DB format. IP
+    """A C extension implementation of a reader for the MaxMind DB format. IP
     addresses can be looked up using the ``get`` method.
     """
 
     closed: bool = ...
 
     def __init__(
-        self, database: Union[AnyStr, int, PathLike, IO], mode: int = MODE_AUTO
+        self,
+        database: Union[AnyStr, int, PathLike, IO],
+        mode: int = ...,
```

Comment:
I noticed this was MODE_AUTO before. But I don't know if there's a real difference.

Reply:
Yeah, I don't know that it matters, but it was probably better documentation before.